wrapanapi/systems/virtualcenter.py

```diff
@@ -432,6 +432,66 @@ def add_disk(self, capacity_in_kb, provision_type=None, unit=None):
                 result = (None, None)
         return result
 
+    def set_cup_hot_add(self, state):
+        """
+        Set cpuHotAdd status (enabled/disabled)
+
+        Args:
+            state (bool): cpuHotAdd state
+
+        Returns:
+            bool : True if already in same state or task ended with success else False
+        """
+        if self.raw.config.cpuHotAddEnabled == state:
+            self.logger.info(
+                "vSphere VM %s cpuHotAddEnabled already in %s state", self.name, state
+            )
+            return True
+
+        spec = vim.vm.ConfigSpec()
+        spec.cpuHotAddEnabled = state
+        task = self.raw.ReconfigVM_Task(spec)
+
+        try:
+            wait_for(lambda: task.info.state not in ["running", "queued"])
+        except TimedOutError:
+            self.logger.exception("Task did not go to success state: %s", task)
+        finally:
+            if task.info.state == "success":
```

Comment:
return task.info.state == "success"

Reply:
Now no need to return as we are using `getter` and `setter`